crates/oci-distribution/src/errors.rs

```diff
@@ -81,6 +81,8 @@ pub enum OciErrorCode {
     Denied,
     /// This operation is unsupported
     Unsupported,
+    /// Too many requests from client
+    Toomanyrequests,
```

Comment:
Can this be `TooManyRequests` to match the casing style of other enums? The serde macro on line 39 will change these all to SCREAMING_SNAKE_CASE. If the casing is incorrect then it will not be cased correctly.

To clarify:

- `Toomanyrequests` -> `TOOMANYREQUESTS`
- `TooManyRequests` -> `TOO_MANY_REQUESTS`

Reply:
I already fiddled with the *SCREAMING_SNAKE_CASE* and, as you said, putting `TooManyRequests` will map to `TOO_MANY_REQUESTS`. In the OCI spec it is `TOOMANYREQUESTS`, so I tricked the screaming snake with the value `Toomanyrequests`.

I hope I understood you correctly, but I couldn't make it work with `TooManyRequests`.

Reply:
So in the PR description I previously mentioned `TOO_MANY_REQUESTS`, which doesn't exist in the spec...